tests/build.gradle

```diff
@@ -16,6 +16,7 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*Java*'
```

Comment:
We could skip the push at the end of the build if we build on Dockerhub directly but attaching our repo via webhook. (Doesn't work ootb with our build processes though)